api/lib/validation/schemaValidator.js

```diff
@@ -0,0 +1,183 @@
+'use-strict'
```

Comment:
Why do we need initial_instance_count? Wouldn't apps already provide this via their deployment manifest?